sql/resource_alert.go

```diff
@@ -0,0 +1,121 @@
+package sql
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go/service/sql"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceAlert() common.Resource {
```

Comment:
Did you consider implementing this in the plugin framework?

Reply:
yes, but we need to have it in the exporter that needs to migrate to the plugin framework as well (until https://github.com/databricks/terraform-provider-databricks/issues/4050 is implemented).

Reply:
yes, but we need to have it in the exporter that needs to migrate to the plugin framework as well (until https://github.com/databricks/terraform-provider-databricks/issues/4050 is implemented).

Reply:
yes, but we need to have it in the exporter that needs to migrate to the plugin framework as well (until https://github.com/databricks/terraform-provider-databricks/issues/4050 is implemented).